packages/store/tests/state.spec.ts

```diff
@@ -1,5 +1,6 @@
 import { TestBed } from '@angular/core/testing';
 
+import { InitState, UpdateState } from '@ngxs/store';
```

Comment:
if we want to import for @ngxs/store we should move all of the imports to there. avoids confusion

Reply:
Ah, those were auto imports. Will move to reference the local files. like the other imports